src/main/java/com/google/api/codegen/ruby/RubyGapicContext.java

```diff
@@ -330,6 +330,10 @@ public String rubyTypeName(TypeRef typeRef) {
     return modules;
   }
 
+  public Iterable<String> getApiModules() {
```

Comment:
Maybe let's rename `getModules` to `getGrpcModules`? This is the distinction, right -- that `getModules` works for types generated by `protoc`, and `getApiModules` works for types generated by GAPIC?